tests/osgi-tests/src/test/java/org/wso2/extension/siddhi/io/http/test/osgi/server/HTTPServer.java

```diff
@@ -0,0 +1,128 @@
+package org.wso2.extension.siddhi.io.http.test.osgi.server;
```

Comment:
Package should come after licence and licence year should be fixed

Reply:
Same for below classes

Reply:
Fixed in 0b31b3775a3e0156cd5309071629ff19e455bc29